l10n_br_fiscal/models/tax.py

```diff
@@ -299,7 +304,8 @@ def _compute_icms(self, tax, taxes_dict, **kwargs):
         # Get Computed IPI Tax
         tax_dict_ipi = taxes_dict.get("ipi", {})
 
-        if partner.ind_ie_dest in (NFE_IND_IE_DEST_2, NFE_IND_IE_DEST_9):
+        if partner.ind_ie_dest in (NFE_IND_IE_DEST_2, NFE_IND_IE_DEST_9) or \
+                cfop.type_move == CFOP_TYPE_MOVE_SALE_INDUSTRY:
```

Comment:
Caso seja uma operação de venda para industrialização o IPI não é adicionado na base de ICMS

Base: Constituição Federal, artigo 155, § 2º, XI e artigo 13, parágrafo 2º da Lei Complementar 87/1996.

Entretanto, integrará a base de cálculo do ICMS se ocorrer qualquer das seguintes condições:

(1) a operação não for realizada entre contribuintes;
(2) o objeto da operação for produto não destinado à industrialização ou à comercialização; e\
(3) a operação não configurar fato gerador de ambos os impostos.

Reply:
Creio que esta validação seja para as CFOPS do tipo "**sale_asset**"

Reply:
Analisamos e realmente isso está incorreto. A condição correta é verificar se a operação é para consumo, independente se o destinatário é contribuinte ou não. Então o correto é considerar também o campo ind_final.

Estamos verificando a necessidade de colocar o campo ind_final na operação de forma a segmentar, por exemplo, a venda para revenda e a venda para consumo.